cms/djangoapps/contentstore/toggles.py

```diff
@@ -61,4 +48,4 @@ def course_import_olx_validation_is_enabled():
     """
     Check if course olx validation is enabled on course import.
```

Comment:
This method should be deprecated because mixing toggles and settings seems like an antipattren.

Reply:
Hmm.. let me think of another way.

Reply:
Moving this to `utils.py`